css/hard-mode-toggle.css

```diff
@@ -0,0 +1,52 @@
+.hide {
+    display: none;
+}
+
+.toggle-container p {
+    text-align: center;
+    color: orange;
+    grid-row: inherit;
+    margin: 10px;
+}
+
+.hard-mode-toggle, .toggle-fill {

```

Comment:
Remove the `toggle-fill` element and instead change the `hard-mode-toggle` background-color to be the dark orange when isEnabled is true.